Changed log-level of ShutDownAssociation from Error to Warning

catap opened the pull request:
When the node has left the cluster, existed nodes throw `akka.remote.transport.Transport$InvalidAssociationException` with message `The remote system terminated the association because it is shutting down`.

This error normally happened when node is leaving the cluster in redeployment proposal and it isn't an error but it creates a noise in monitoring/alert system.

So, I propose to log it as Warning.

lightbend-cla-validator commented:
Hi @catap,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

raboof commented:
OK TO TEST

catap commented:
@raboof fixed formating.

johanandren commented on akka-remote/src/main/scala/akka/remote/Endpoint.scala:
```diff
@@ -556,7 +556,8 @@ private[remote] class EndpointWriter(
 
   override val supervisorStrategy = OneForOneStrategy(loggingEnabled = false) {
-    case NonFatal(e) ⇒ publishAndThrow(e, Logging.ErrorLevel)
+    case e: ShutDownAssociation ⇒ publishAndThrow(e, Logging.WarningLevel)
```
It is even just info? It happens on leaving, and that isn't necessarily something bad.

catap replied:
@johanandren it is a good point. Thanks. Updated.

catap committed:
Changed log-level of ShutDownAssociation from Error to Warning

When the node has left the cluster, existed nodes throw
akka.remote.transport.Transport$InvalidAssociationException with message `The
remote system terminated the association because it is shutting down`.

This error normally happened when node is leaving the cluster in redeployment
proposal and it isn't an error but it creates a noise in monitoring/alert system.

So, I propose to log it as Warning.

johanandren reviewed (approved):
LGTM, but would be good to have an opinion from @patriknw

patriknw reviewed (approved):
LGTM, should be ok
